core/templates/dev/head/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -40,6 +40,9 @@ oppia.controller('SettingsTab', [
       EXPLORATION_TITLE_INPUT_FOCUS_LABEL);
 
     $scope.CATEGORY_LIST_FOR_SELECT2 = [];
+    //Empty category added to force dropdown to scroll top
```

Comment:
This code does the trick. the question is now, when we add a new category in the dropdown, it is appended to the list. We could add them to the top of the list so we dont have to distinguish between empty or non-empty categories (we would always unshift the array instead of pushing new elements).

`        if (!categoryIsInSelect2) {
          if (explorationCategoryService.savedMemento) {
            $scope.CATEGORY_LIST_FOR_SELECT2.push({
              id: explorationCategoryService.savedMemento,
              text: explorationCategoryService.savedMemento
            });
          } else {
            $scope.CATEGORY_LIST_FOR_SELECT2.unshift({
              id: explorationCategoryService.savedMemento,
              text: explorationCategoryService.savedMemento
            });
          }
        }
`